scene/2d/camera_2d.cpp

```diff
@@ -77,6 +77,9 @@ void Camera2D::_update_process_mode() {
 }
 
 void Camera2D::set_zoom(const Vector2 &p_zoom) {
+	// Setting zoom to zero causes 'affine_invert' issues
+	ERR_FAIL_COND_MSG(Math::is_zero_approx(p_zoom.x) || Math::is_zero_approx(p_zoom.y), "Zoom level must be different than 0 (can be negative).");
```

Comment:
"different from 0" sounds more natural to me (though I checked and both seem to be valid).

Reply:
Fixed!